src/NServiceBus.Core/Transports/Msmq/MsmqQueueCreator.cs

```diff
@@ -7,12 +7,16 @@ namespace NServiceBus.Transports.Msmq
     using Logging;
     using Support;
 
+    static class MsmqConstants
```

Comment:
@SimonCropp I wanted to reuse those constants elsewhere, and making them internal within `QueueCreator` didn't feel right.